test/node.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var assert = require('simple-assert');
+var type = require('..');
+(function () {
+  if (typeof process === 'undefined' || typeof process.release !== 'object' || !process.release.name) {
+    return describe.skip;
+  }
+  return describe;
+}())('Node Specific', function () {
```

Comment:
@keithamus : I loved what you did with `itIf()` and wondered if you wouldn't want to do the same thing here to have a more clear syntax (also for the sake of consistency). Something like:

```javascript
function describeIf(condition) {
  return condition ? describe : describe.skip;
}
function isNode() {
  return typeof process === 'undefined' || typeof process.release !== 'object' || !process.release.name;
}
describeIf(isNode())('Node Specific', function () {
```
It's 1 more line of code but the describe-block itself reads like a textbook.